bouncer/src/repo/manipulator/repo_manipulator.cpp

```diff
@@ -829,7 +829,9 @@ const repo::manipulator::modelconvertor::ModelImportConfig *config)
 
 	if (!repo::manipulator::modelconvertor::AssimpModelImport::isSupportedExts(fileExt) 
```

Comment:
Merge this check with lines 842-844:

Something like: 
```suggestion
	bool useAssimp = repo::manipulator::modelconvertor::AssimpModelImport::isSupportedExts(fileExt) 
	bool useIFCImporter = fileExt == ".IFC" && (!config || config->getUseIFCOpenShell());
	bool useRepoImporter = fileExt == ".BIM";
	bool useOdaImporter = repo::manipulator::modelconvertor::OdaModelImport::isSupportedExts(fileExt);
	
	if(!(useAssimp || useIFCImporter || useRepoImporter || useOdaImporter) {
		//....
	}
```